examples/simple-nft-minter/index.mjs

```diff
@@ -0,0 +1,43 @@
+import { loadStdlib } from '@reach-sh/stdlib';
+const stdlib = loadStdlib(process.env);
+
+const startingBalance = stdlib.parseCurrency(1000);
+
+const [minter, receiver] = await stdlib.newTestAccounts(2, startingBalance);
+
+const gasLimit = 5000000;
+minter.setGasLimit(gasLimit);
+receiver.setGasLimit(gasLimit);
+
+const mintAddr = minter.getAddress();
+console.log(`Your address is ${mintAddr}`);
+const recAddr = receiver.getAddress();
+console.log(`receiver's address is ${recAddr}`);
+
+const minterAddrFormat = await stdlib.formatAddress(minter);
+console.log(`The minter's formatted address is ${minterAddrFormat}`);
+const receiverAddrFormat = await stdlib.formatAddress(receiver);
+console.log(`The receiver's formatted address is ${receiverAddrFormat}`);
+
+const fmt = (x) => stdlib.formatCurrency(x, 4);
+const getBal = async (who) => fmt(await stdlib.balanceOf(who));
+const bal = await getBal(minter);
+console.log(`Minter starting balance: ${bal}.`);
+
+console.log(`Creating the NFT`);
+const theNFT = await stdlib.launchToken(minter, "nftName", "SYM", { supply: 1, url: "https://reach.sh", clawback: null, freeze: null, defaultFrozen: false, reserve: null, note: Uint8Array[1]});
```

Comment:
This is formatted ugly and it does not match the ARC specs for the metadata. You should confirm that whatever you do will show up in the testnet galleries.

Reply:
You should make this have cute names that "feel" like a real NFT.